### PR TITLE
Try to fix flaky EventRouter e2e test

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### Description
Try to fix flaky EventRouter e2e test. Add code for waiting elasticsearch deployment will be ready. 

/cc 
/assign @jcantrill 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA:
- Enhancement proposal:

### CODE REVIEW

File: hack/testing-olm/test-040-eventrouter.sh [hunk 1]
```diff
@@ -57,6 +57,25 @@ deploy_eventrouter() {
     fi
 }
 
+wait_elasticsearch() {
+  local looptries=${MAX_DEPLOY_WAIT_SECONDS}
```
Comment (jcantrill) [inline hack/testing-olm/test-040-eventrouter.sh:61]: this is seconds but the looping is simply an ordinal number.  We should either:
* limit the number of tries to a max
* or keep track of time and continue until that time is exceeded

Comment (vparfonov) [reply hack/testing-olm/test-040-eventrouter.sh:61]: i did it similar to the https://github.com/openshift/cluster-logging-operator/blob/master/hack/testing-olm/test-040-eventrouter.sh#L43

**Review by ajaygupta978 — APPROVED** [review]

**Review by ajaygupta978 — APPROVED** [review]

**Comment by vimalk78** [comment]
@vparfonov can you use `oc wait` ? then may be you can avoid the loop

```
$ oc wait -h
Experimental: Wait for a specific condition on one or many resources.

 The command takes multiple resources and waits until the specified condition is seen in the Status field of every given
resource.

 Alternatively, the command can wait for the given set of resources to be deleted by providing the "delete" keyword as
the value to the --for flag.
```

**Review by vimalk78 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**